trace-resources/src/Cardano/Logging/Resources/Types.hs

```diff
@@ -118,12 +118,12 @@ instance LogFormatting ResourceStats where
       ]
 
     asMetrics rs =
-      [ IntM "stat.cputicks" (fromIntegral $ rCentiCpu rs)
-      , IntM "mem.resident" (fromIntegral $ rRSS rs)
-      , IntM "rts.gcLiveBytes" (fromIntegral $ rLive rs)
-      , IntM "rts.gcMajorNum" (fromIntegral $ rGcsMajor rs)
-      , IntM "rts.gcMinorNum" (fromIntegral $ rGcsMinor rs)
-      , IntM "rts.gcticks" (fromIntegral $ rCentiGC rs)
-      , IntM "rts.mutticks" (fromIntegral $ rCentiMut rs)
-      , IntM "rts.stat.threads" (fromIntegral $ rThreads rs)
+      [ IntM "Resources.Stat.Cputicks" (fromIntegral $ rCentiCpu rs)
+      , IntM "Resources.mem.Resident" (fromIntegral $ rRSS rs)
```

Comment:
@jutaro Is it `"Resources.mem.Resident"` and not `"Resources.Mem.Resident"`?

Reply:
Fixed it

Reply:
Should be: Resources.Mem.Resident

Reply:
Thanks